search-ui/src/resources/contact-info/registries-info.ts

```diff
@@ -9,9 +9,9 @@ export const RegistriesInfo: ContactInfoI[] = [
   },
   {
     icon: 'mdi-phone',
-    href: 'tel:+1-250-952-0568',
+    href: 'tel:+1-250-387-7848',
```

Comment:
I noticed that this is super out of date. If you look at the designs (https://preview.uxpin.com/4648cf2627971eb4281b7ccc7c1258224b3d8084#/pages/164129752/simulate/sitemap) and what we have in other repos, this is the number being used for "Victoria Office".

Reply:
Good to know -- I will double check the other UIs BTR is managing